cpu/cc26xx-cc13xx/cc26xx.ld

```diff
@@ -90,11 +90,15 @@ SECTIONS
         _ebss = .;
     } > SRAM
 
+    _end = .;  /* End of the .bss segment. */
+
     .ccfg :
     {
         KEEP(*(.ccfg))
     } > FLASH_CCFG
 
+    _stack = .;  /* This symbol is used by the stack check library. */
```

Comment:
I revisited this PR now on our custom cc13xx board, but got problem booting with _stack defined here. I moved it before .ccfg (as I noticed you do in contiki-ng) and it worked fine. Not versed in linker scripts but looking at the values it yields it seems strange to me that it should work at all as is?

Reply:
You're right: here, `_stack` has its VMA set to flash memory, so there is no way it can work.